src/app/zap-templates/zcl/zcl.json

```diff
@@ -641,6 +642,16 @@
         "Power Topology": ["FeatureMap"],
         "Valve Configuration and Control": ["RemainingDuration"],
         "Boolean State Configuration": ["CurrentSensitivityLevel"],
+        "Water Heater Management": [
+            "HeaterTypes",
```

Comment:
This lines should be removed from this PR.

Reply:
Done